fedimint-cli/src/lib.rs

```diff
@@ -320,6 +321,15 @@ async fn load_or_generate_mnemonic(db: &Database) -> Result<Mnemonic, CliError>
     )
 }
 
+fn generate_random_password() -> String {
+    const PASSWORD_LENGTH: usize = 16;
+    thread_rng()
+        .sample_iter(&Alphanumeric)
```

Comment:
We should choose one case (upper or lower) and remove characters that look similar (`1`, `I`, `l`, …).